CHANGELOG.md

```diff
@@ -1,5 +1,188 @@
 # Changelog
 
+## 🔎 1.1.0
+
+Wrangler 1.1.0 includes a number of improvements to documentation and project stability, including: 
+
+- ### Features
+
+  - **Add message module for terminal output - [ashleymical], [issue/219] [pull/263]**
+
+    We've made improvements to Wrangler's terminal output functionality, with support for various log levels and implementations in Wrangler's API for easily using the log levels in future development.
+
+    [ashleymical]: https://github.com/ashleymichal
+    [issue/219]: https://github.com/cloudflare/wrangler/issues/219
+    [pull/263]: https://github.com/cloudflare/wrangler/pull/263
+
+  - **Adds more descriptive subdomain errors - [EverlastingBugstopper], [issue/207] [pull/259]**
+
+    Warns users when attempting to add a subdomain w/o account_id in wrangler.toml
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/207]: https://github.com/cloudflare/wrangler/issue/207
+    [pull/259]: https://github.com/cloudflare/wrangler/pull/259
+
+  - **Allow custom webpack configuration in wrangler.toml - [EverlastingBugstopper], [issue/246] [pull/253]**
+
+    Users can now specify custom webpack configuration filename, via a `webpack_config` key in `wrangler.toml`.
```

Comment:
we'll want to link to/have docs on this

Reply:
This PR included an update to the README